audiobook/main.py

```diff
@@ -82,6 +83,8 @@ def create_json_book(self, input_book_path, password=None):
             json_book, metadata = mobi_to_json(input_book_path)
         elif input_book_path.startswith("http"):
             json_book, metadata = html_to_json(input_book_path)
+        elif input_book_path.endswith(".docx") or input_book_path.endswith(".doc"):
```

Comment:
You can write it like this. if file_name.endswith(tuple(ext1, ext2)):
It will look good

Reply:
Thanks, fixed & pushed.

Reply:
Okay, fixed & pushed. I've tested this locally and it worked, but it's updated to match your suggestion exactly now.